README.md

```diff
@@ -31,8 +31,8 @@ $ git clone https://github.com/catalystneuro/nwb-conversion-tools
 $ cd nwb-conversion-tools
 $ conda env create -f make_env.yml
 $ conda activate nwb_conversion_env
-$ pip install .
```

Comment:
I removed this line because it is unnecessary

Reply:
In a separate PR, can you split this section into two options? Option 1, conda install via environment and activation, Option 2, install in current environment via `pip` (and the current PyPI release).